cplusplus/include/vips/VImage8.h

```diff
@@ -2977,9 +2977,9 @@ static VImage heifload_source( VSource source, VOption *options = 0 );
  *   - **Q** -- Q factor, int.
  *   - **lossless** -- Enable lossless compression, bool.
  *   - **compression** -- Compression format, VipsForeignHeifCompression.
+ *   - **speed**: -- CPU effort, 0 slowest - 8 fastest, AV1 compression only, int.
+ *   - **subsample_mode** -- Select chroma subsample operation mode, VipsForeignSubsample.
  *   - **strip** -- Strip all metadata from image, bool.
- *   - **background** -- Background value, std::vector<double>.
```

Comment:
I think `background` and `page_height` might have been copypasta as they are not part of heifsave.

Reply:
They are defined (and implemented) in the base class for all savers. We should document them in all savers or none ... I guess none is simpler.

Reply:
I've kept them in for now, we can remove these in a later PR if required.